packages/apputils/src/sessioncontext.tsx

```diff
@@ -196,6 +197,11 @@ export interface ISessionContext extends IObservableDisposable {
    */
   readonly prevKernelName: string;
 
+  /**
+   * The kernel manager
+   */
+  readonly kernelManager: Kernel.IManager;
```

Comment:
Should this be made optional?

It looks like by adding this to the interface, downstream code that "implements ISessionContext" will now have to provide a `kernelManager`, which may be considered as breaking.

Reply:
Although there is likely not many downstreams implementing their own `ISessionContext`, so another option could also be to document this change in https://jupyterlab.readthedocs.io/en/latest/extension/extension_migration.html for visibility.

Reply:
I did a web search and couldn't find any public instance of `implements ISessionContext` outside of JupyterLab, but you're totally right about the actual API contract. I've updated the code.